tutorial-template.md

```diff
@@ -0,0 +1,125 @@
+        * Include a 'More Info', 'Resources', or 'See Also' section
+    * AVOID:
+        * Notes, Tips, and Important boxes. Users find them distracting and generally skip over them. 
+        * Excessive bulleted or numbered lists. Use only as needed for clarity. 
+
+-->
+
+# Tutorial: <verb and action> 
+ <!-- (example) "Tutorial: Write and Simulate Qubit-level programs in Q#". This should match the title in the metadata-->
+
+ <!-- 
+    * Provide a high-level abstract of the tutorial: what the user will do, accomplish, and learn. "In this tutorial you will do..." 
+
+    * Provide a bullet list of what is covered, based on the H2 procedure steps. Use the 'checklist' class to display green checkmarks (for this checklist only)
+-->
+
+> [!div class="checklist"]
+> * Procedure-1
+> * Procedure-2
+> * Procedure-3
+
+## Prerequisites
+ <!-- 
+ Provide as necessary:
+     * Software versions of installations
+     * Previous tutorials or knowledge
+     * Preparations such as creating a host project (this can be a link to How to Create a Q# Project)
+-->
+
+## Procedure_1 (Build the Scenario)
+<!-- "This procedure walks you through writing the code to..."  -->
+
+<!-- Include:
+    * A short sentence or two describing what happens in this section
+    * List and link the steps performed in this section
+    * [optional] Display the finished and commented Q# code block. It can be beneficial to see the big picture first. 
+-->
+
+### Step_1 
+<!-- Step names should describe an action, for example, "Add the Quantum.Operations namespace definition", "Measure the qubits and store the results">
+<!-- Include
+    * Brief description of what this step does
+    * Individual steps as needed
+    * Explain what this step accomplishes, what commands were used
+    * Other pertinent notes or comments
+-->
+### Step_2
+
+### Step_3
+
+## Procedure_2 (Run the Scenario)
+<!-- "This procedure walks you through running the code to..."  -->
+
+<!-- Include:
+    * A short sentence or two describing what happens in this section
+    * (If more than 2 or 3 steps) List and link the steps performed in this section
+    * A tabbed structure with code examples for the Python (if supported for the scenario), C# for VS, and C# for VS Code hosts'
+-->
+### Step_1
+<!-- Include (as needed)
+    * Brief description of what this step does
+    * Individual steps as needed
+    * Explain what this step accomplishes, what commands were used
+    * Other pertinent notes or comments
+-->
+### Step_2
+
+## Procedure_3 (Results)
+<!-- "This procedure analyzes the output of the scenario..."  -->
+
+<!-- Display the results and explain how they relate to the code and concept 
+
+     Create output results for each of the host code examples used in Procedure 2.
+
+-->
+
+## Procedure_4 (Variations and exercises)
```

Comment:
Remove variations and exercises section. A tutorial -- as our doc group defines - show one prescriptive way to accomplish a scenario.

Reply:
Variations could be a tutorial doc listed under "Next steps"